Makefile

```diff
@@ -119,7 +119,7 @@ build-linux-arm64:
 	GOOS=linux GOARCH=arm64 LEDGER_ENABLED=false $(MAKE) build
 
 $(BUILD_TARGETS): go.sum $(BUILDDIR)/
-	cd ${CURRENT_DIR}/simapp && go $@ -mod=readonly $(BUILD_FLAGS) $(BUILD_ARGS) ./...
+	cd ${CURRENT_DIR}/simapp && go $@ -mod=ErrReadOnly $(BUILD_FLAGS) $(BUILD_ARGS) ./...
```

Comment:
please revert (and the rest with that change too)

Reply:
omg.   you got it!